gadgets/src/algorithms/prf/blake2s.rs

```diff
@@ -459,6 +461,16 @@ impl<F: PrimeField> AllocGadget<[u8; 32], F> for Blake2sOutputGadget {
     }
 }
 
+impl<F: PrimeField> ToBitsBEGadget<F> for Blake2sOutputGadget {
+    fn to_bits_be<CS: ConstraintSystem<F>>(&self, cs: CS) -> Result<Vec<Boolean>, SynthesisError> {
+        self.0.to_bits_le(cs)
+    }
+
+    fn to_bits_be_strict<CS: ConstraintSystem<F>>(&self, cs: CS) -> Result<Vec<Boolean>, SynthesisError> {
+        self.0.to_bits_le_strict(cs)
```

Comment:
@weikengchen can you confirm this `to_bits_be_strict` method should be returning `self.0.to_bits_le_strict()`?